scripts/reboot

```diff
@@ -9,7 +9,7 @@ REBOOT_TIME=$(date)
 VMCORE_FILE=/proc/vmcore
 if [ -e $VMCORE_FILE -a -s $VMCORE_FILE ]; then
         echo "We have a /proc/vmcore, then we just kdump'ed"
-        echo "User issued 'kdump' command [User: kdump, Time: ${REBOOT_TIME}]" > ${REBOOT_CAUSE_FILE}
+        echo "Kernel Panic (Time: ${REBOOT_TIME})" > ${REBOOT_CAUSE_FILE}
```

Comment:
Suggest changing parentheses to square brackets to match the message format produced by the CLI.